### ISSUE TITLE
DWD download fails due to files specified which aren't in the data directory

### ISSUE THREAD

**Issue by tom-andersson** [jonas-scholz123/sim2real-downscaling#1]
`python3 sim2real/preprocess/dwd.py` fails because it specifies some file names that aren't present in the [DWD data directory](https://opendata.dwd.de/climate_environment/CDC/observations_germany/climate/hourly/air_temperature/recent/), names the first two PDFs here:
https://github.com/jonas-scholz123/sim2real-downscaling/blob/0161881d6034026a0a5789eae9d0ff458518a423/sim2real/preprocessing/dwd.py#L18-L19

Deleting the PDF lines allowed the download to run for me.

To guard against file name changes it could be better to get `urllib` to just download all files in the folder rather than specifying the file names. I'm not sure how to do this but I assume it would be straightforward. Although the cybersecurity of that is another issue...

**Comment by jonas-scholz123** [comment]
You're right, that is annoying. Very low priority until I've got time - this was a quick and dirty way to get data!

**Comment by tom-andersson** [comment]
Definitely very low priority!

**Comment by tom-andersson** [comment]
Also low priority, but I presume this should by default be the `historical/` rather than `recent/` directory, otherwise only the last couple years of data is downloaded: https://github.com/jonas-scholz123/sim2real-downscaling/blob/0161881d6034026a0a5789eae9d0ff458518a423/sim2real/preprocessing/dwd.py#L528
